### ISSUE TITLE
Struggling to install later versions... something about the required versions of django-mptt

### ISSUE THREAD

**Issue by mkoistinen** [django-cms/django-filer#477]
``` bash
/var/www/press$ pip install https://github.com/stefanfoulis/django-filer/archive/develop.zip
Downloading/unpacking https://github.com/stefanfoulis/django-filer/archive/develop.zip
  Downloading develop.zip (unknown size): 1.1Mb downloaded
  Running setup.py egg_info for package from https://github.com/stefanfoulis/django-filer/archive/develop.zip

Requirement already satisfied (use --upgrade to upgrade): Django>=1.4 in /home/django/.virtualenvs/press/lib/python2.7/site-packages (from django-filer==0.9.9a1.dev1)
Requirement already satisfied (use --upgrade to upgrade): easy-thumbnails>=1.0 in /home/django/.virtualenvs/press/lib/python2.7/site-packages (from django-filer==0.9.9a1.dev1)
Downloading/unpacking django-mptt==0.5.2,==0.6,==0.6.1 (from django-filer==0.9.9a1.dev1)
  Could not find a version that satisfies the requirement django-mptt==0.5.2,==0.6,==0.6.1 (from django-filer==0.9.9a1.dev1) (from versions: )
No distributions matching the version for django-mptt==0.5.2,==0.6,==0.6.1 (from django-filer==0.9.9a1.dev1)
Storing complete log in /home/django/.pip/pip.log

/var/www/press$ pip freeze | grep django-mptt
django-mptt==0.6.1
```

PyPI version 0.9.5 installs fine, 0.9.8 and develop does not.

**Comment by kaesemeister** [comment]
I applied the fix to version 0.9.8
`pip install git+https://github.com/nadobit/django-filer.git@support/0.9.8`

**Comment by yakky** [comment]
0.9.9 has been released!
